Update based on legal requirements. (#900)
huggingface/optimum-habana: docs/source/index.mdx

-->

> # 🤗 Optimum Intel(R) Gaudi(R) accelerator

Reviewer: ```suggestion
# 🤗 Optimum for Intel(R) Gaudi(R) accelerator
```

> # 🤗 Optimum for Intel® Gaudi® accelerator

Reviewer 2: ```suggestion
# Optimum for Intel Gaudi
```

> 🤗 Optimum for Intel® Gaudi® accelerator is the interface between the 🤗 Transformers and 🤗 Diffusers libraries and [Intel® Gaudi® accelerator Gaudi processor (HPU)](https://docs.habana.ai/en/latest/index.html).

Reviewer 2: ```suggestion
Optimum for Intel Gaudi is the interface between the Transformers and Diffusers libraries and [Intel® Gaudi® AI Accelerators (HPUs)](https://docs.habana.ai/en/latest/index.html).
```

> 🤗 Optimum Intel(R) Gaudi(R) accelerator is the interface between the 🤗 Transformers and 🤗 Diffusers libraries and [Intel(R) Gaudi(R) accelerator Gaudi processor (HPU)](https://docs.habana.ai/en/latest/index.html).

Reviewer: ```suggestion
🤗 Optimum for Intel(R) Gaudi(R) accelerator is the interface between the 🤗 Transformers and 🤗 Diffusers libraries and [Intel(R) Gaudi(R) accelerator (HPU)](https://docs.habana.ai/en/latest/index.html).
```

It provides a set of tools that enable easy model loading, training and inference on single- and multi-HPU settings for various downstream tasks as shown in the table below.

HPUs offer fast model training and inference as well as a great price-performance ratio.

> Check out [this blog post about BERT pre-training](https://huggingface.co/blog/pretraining-bert) and [this article benchmarking Habana Gaudi2 versus Nvidia A100 GPUs](https://huggingface.co/blog/habana-gaudi-2-benchmark) for concrete examples.

Reviewer 2: ```suggestion
Check out [this blog post about BERT pre-training](https://huggingface.co/blog/pretraining-bert) and [this post benchmarking Intel Gaudi 2 with NVIDIA A100 GPUs](https://huggingface.co/blog/habana-gaudi-2-benchmark) for concrete examples.
```

If you are not familiar with HPUs, we recommend you take a look at [our conceptual guide](./concept_guides/hpu).

> The following model architectures, tasks and device distributions have been validated for 🤗 Optimum for Intel® Gaudi® accelerator:

Reviewer 2: ```suggestion
The following model architectures, tasks and device distributions have been validated for Optimum for Intel Gaudi:
```

> The following model architectures, tasks and device distributions have been validated for 🤗 Optimum Intel(R) Gaudi(R) accelerator:

Reviewer: ```suggestion
The following model architectures, tasks and device distributions have been validated for 🤗 Optimum for Intel(R) Gaudi(R) accelerator:
```
